js/blocks/loader/editor.scss

```diff
@@ -242,6 +242,24 @@ div[class^="wp-block-block-lab-"],
   }
 }
 
+.bl-fetch-input {
+  >.components-base-control__field {
+    display: flex;
```

Comment:
It's nice how the plugin has an `.scss` build system, even though there aren't many styles. It keeps this more DRY.